cmd/localCI/repo.go

```diff
@@ -260,10 +260,23 @@ func (r *Repo) loop() {
 
 	appendPullRequests := func(revisions *[]revision, prs []int) error {
 		for _, prNumber := range prs {
+			var pr revision
+			var err error
+
 			r.logger.Debugf("requesting pull request %d", prNumber)
-			pr, err := newPullRequest(prNumber, r.prConfig)
```

Comment:
just out of curiosity, why did you change short variable declaration?

Reply:
```newPullRequest``` returns an object of type ```pullRequest``` that can be assigned to an object of type ```revision``` but not vice-versa ("polymorphism"). In the line 274 I look for an object of type ```revision``` with the same ```prNumber``` and assign it to ```rev``` then in the line 279 I copy ```rev``` (type ```revision```) to ```pr``` (type ```revision```) because ```pr``` is appended to a map of ```revision```s. 
In conclusion if the type of ```pr``` was ```pullRequest``` I couldn't assign it to ```rev``` hence I couldn't copy it into the map (```revisionsTested```)